src/components/Slideover/slideover.scss

```diff
@@ -11,13 +11,16 @@
   box-sizing: border-box;
   height: 100%;
   max-height: calc(100vh - 40px);
-  max-width: 624px;
-  min-width: 400px;
   margin-left: auto;
   padding: 20px;
   background-color: $white;
   border-radius: 5px;
 
+  @media (min-width: $desktops-large) {
```

Comment:
I think you can't use `$desktops-large` because this variable is not part of the DS but it's from the storyfront project:

https://github.com/storyblok/storyfront/blob/main/src/assets/styles/variables.scss#L12